addons/html_editor/static/tests/test_helpers/user_actions.js

```diff
@@ -209,19 +209,19 @@ export function setColor(color, mode) {
 }
 
 // Mock an paste event and send it to the editor.
-export async function pasteData(editor, text, type) {
-    const mockEvent = {
-        dataType: "text/plain",
-        data: text,
+function pasteData(editor, text, type) {
```

Comment:
I think you can trigger a "real" paste event instead. This is a snippet for mocking a "copy" event, I am assuming something similar would work for "paste" events:

```js
                    const clipboardData = new DataTransfer();
                    clipboardData.setData('text/plain', 'should stay');
                    await triggerEvent(editor.editable, 'copy', { clipboardData });
```